CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/startup/ConstellationLAFSettings.java

```diff
@@ -0,0 +1,502 @@
+            tabDarkShadow = new Color(20, 25, 30);
+            inactiveTabBackground = new Color(40, 55, 90);
+        } else {
+            // Fixed set of Blue shade colors
+            // appropriate for standard Metal LAF
+            activeTabBackground = new Color(130, 170, 255);
+            tabDarkShadow = new Color(220, 235, 250);
+            inactiveTabBackground = new Color(170, 195, 230);
+        }
+        
+        try {
+            final Class<?> tabDisplayer = (Class<?>) UIManager.getDefaults()
+                    .get("org.netbeans.swing.tabcontrol.plaf.MetalViewTabDisplayerUI");
+            
+            if (tabDisplayer != null) {
+                // get Reflection objects before updating any UI settings
+                final Field actBgCol = tabDisplayer.getDeclaredField("actBgColor");
+                final Field inactBgCol = tabDisplayer.getDeclaredField("inactBgColor");
+
+                // Update the UIManager with settings appropriate for Metal LAF
+                UIManager.getDefaults().put("control", inactiveTabBackground);
+                UIManager.getDefaults().put("controlHighlight", activeTabBackground);
+                UIManager.getDefaults().put("controlDkShadow", tabDarkShadow);
+                UIManager.getDefaults().put("TabRenderer.selectedActivatedBackground", activeTabBackground);
+                UIManager.getDefaults().put("TabRenderer.selectedBackground", inactiveTabBackground);
+                
+                // reset static field "actBgColor" value to activeTabBackground
+                actBgCol.setAccessible(true); //NOSONAR
+                actBgCol.set(tabDisplayer, activeTabBackground); //NOSONAR
+
+                // reset static field "inactBgColor" value to inactiveTabBackground
+                inactBgCol.setAccessible(true); //NOSONAR
+                inactBgCol.set(tabDisplayer, inactiveTabBackground); //NOSONAR
+            } else {
+                // Unable to apply changes to Metal LAF
+                // Possibly caused by a change in a newer NetBeans version
+                LOGGER.info(" >>> Unable to apply Tab coloring to Metal LAF : org.netbeans.swing.tabcontrol.plaf.MetalViewTabDisplayerUI not defined.");
+                ouputUIDefaultValues(null, null);
+            }
+        } catch (final IllegalAccessException | IllegalArgumentException | 
+                NoSuchFieldException | SecurityException e) {
+            // An exception here would indicate a change in the internal LAF 
+            // packages provided by NetBeans
+            final String errorMessage = " >>> Error applying Metal Laf colors : " + e.toString();
+            LOGGER.log(Level.SEVERE, errorMessage, e);
+        }
+    }
+
+    /**
+     * Convenience method to assist with identifying UImanager settings
+     * 
+     * @param keyFilter limit output to key entries containing the keyFilter
+     * @param valueFilter limit output to values containing the valueFilter
+     */
```

Comment:
is this method still required? If not are you able to remove it?

Reply:
Technically not required, but I'd like it to be in for a few releases, as there may be other Windows environments to cater for, and this will be the only means of getting the information we need.

Reply:
Ok. In that case, are you able to mention that in the javadoc for the function? Just so that is clear

Reply:
Done